### ISSUE TITLE
Request: Bulk Add (import), Bulk Copy (export) magnet links

### ISSUE THREAD

**Issue by a-raccoon** [rakshasa/rtorrent#935]
The client qBittorrent allows you to paste hundreds of magnet links (or info-hashes or http links) in a single `Add Torrent(s)` operation.  It also allows you to select multiple loaded torrents to `Copy Magnet Link(s)` to clipboard for export to another client or friend.

I would really love to bulk export magnet links from qBittorrent then bulk import them into ruTorrent in one single click, instead of 4500 clicks.

**Comment by rakshasa** [comment]
This seems like it should be done through an external utility or web ui.

**Comment by chros73** [comment]
> This seems like it should be done through an external utility 

Exactly: you can use a [script like this](https://github.com/chros73/rtorrent-ps-ch_setup/blob/master/ubuntu-14.04/home/chros73/bin/addMagnetLinksAria2.sh) to add multiple magnet links (converting them into fake torrent files) using aria2.

**Comment by realyukii** [comment]
is there a way to bulk add import torrent files instead of magnet links? I have new machine that I want to make it as seeder and need to import exported torrent files from qBittorrent client in bulk

**Comment by rakshasa** [comment]
Should be relatively easy to add a command that splits up a load command by newline and loads each.

**Comment by realyukii** [comment]
ah sorry, I just read the wiki, and previously unaware of wildcard as an option

**Comment by rakshasa** [comment]
Yeah, that's also an option for normal torrents.
